backend/estimate-service/pom.xml

```diff
@@ -0,0 +1,120 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.egov</groupId>
+    <artifactId>estimate-service</artifactId>
+    <packaging>jar</packaging>
+    <name>estimate-service</name>
+    <version>1.0.0</version>
+    <properties>
+        <java.version>1.8</java.version>
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+        <maven.compiler.target>${java.version}</maven.compiler.target>
+        <postgresql.version>42.3.3</postgresql.version>
+    </properties>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>1.5.9.RELEASE</version>
```

Comment:
Upgrade to Spring 2. 
Check the tracer library for `minor` version compatibility.

Reply:
I have upgraded it to "2.2.6.RELEASE" version of Spring boot (same as tracer library's version)